src/circuit/gadget/ecc/chip/mul.rs

```diff
@@ -62,17 +66,34 @@ pub struct Config {
     overflow_config: overflow::Config,
 }
 
-impl From<&EccConfig> for Config {
-    fn from(ecc_config: &EccConfig) -> Self {
+impl Config {
+    pub(super) fn configure(
+        meta: &mut ConstraintSystem<pallas::Base>,
+        add_config: add::Config,
+        lookup_config: LookupRangeCheckConfig<pallas::Base, { sinsemilla::K }>,
+        advices: [Column<Advice>; 10],
+    ) -> Self {
+        let hi_config = incomplete::Config::configure(
+            meta, advices[9], advices[3], advices[0], advices[1], advices[4], advices[5],
+        );
+        let lo_config = incomplete::Config::configure(
+            meta, advices[6], advices[7], advices[0], advices[1], advices[8], advices[2],
+        );
```

Comment:
I know this hasn't changed from before this PR, but what's the motivation for this ordering?

Reply:
I think it evolved organically as we optimised the chip? It does look a bit worse than it is because of the argument ordering to `configure()`.